scripts/postbuild.ts

```diff
@@ -0,0 +1,48 @@
+/*
+  This script prepares the central `build` directory for NPM package creation.
+  It first copies all non-code files into the `build` directory, including `package.json` which
+  is edited to adjust entry point paths. These corrections are performed so that they align with
+  the directory structure inside `build`.
```

Comment:
```suggestion
  This script prepares the central `build` directory for NPM package creation.
  It first copies all non-code files into the `build` directory, including `package.json`, which
  is edited to adjust entry point paths. These corrections are performed so that the paths align with
  the directory structure inside `build`.
```